tools/webdriver/webdriver/client.py

```diff
@@ -581,7 +581,7 @@ def source(self):
         return self.send_session_command("GET", "source")
 
     @command
-    def new_window(self, type_hint=None):
+    def new_window(self, type_hint="window"):
```

Comment:
Can we just use `tab` here? It's less resource intensive than windows and all browsers actually support it.